Services/Simulations.cs

```diff
@@ -414,7 +442,7 @@ public async Task<bool> TryToSetDeviceDeletionCompleteAsync(string simulationId)
             return await this.TryToUpdateSimulation(simulation);
         }
 
-        private async Task<bool> TryToUpdateSimulation(Models.Simulation simulation)
+        public async Task<bool> TryToUpdateSimulation(Models.Simulation simulation)
```

Comment:
no need to expose this method, which was created only to support the public one. `UpsertAsync` should be the one to use

Reply:
pls add the missing `Async` suffix to the method name

Reply:
updated